models/company.py

```diff
@@ -0,0 +1,52 @@
+from db import db
+# import enum
+#
+#
+# class BusinessChoices(enum.Enum):
+#     finance = 'finance'
+#     health_care = 'health care'
+#     education = 'education'
+
+class CompanyModel(db.Model):
+    __tablename__ = 'company'
+
+    id = db.Column(db.Integer, primary_key=True)
+    us_employer_id = db.Column(db.String(50))
+    company_name = db.Column(db.String(200))
+    # business_area = db.Column(db.Enum(BusinessChoices), nullable=False)  # ******* this should be enum *******
+    business_area = db.Column(db.String(50), nullable=False)
+    number_of_employees = db.Column(db.Integer)
+    phone = db.Column(db.String(11))
```

Comment:
we don't need to restrict sizes of phone fields. There is no enforcement in verification of phone number format so let's accept whatever string the clients send us

Reply:
oh sorry, forget it, I saw you restricted in validation function